groups/ntc/ntcr/ntcr_streamsocket.cpp

```diff
@@ -6433,6 +6095,24 @@ ntsa::Error StreamSocket::setReadQueueWatermarks(bsl::size_t lowWatermark,
     return ntsa::Error();
 }
 
+ntsa::Error StreamSocket::timestampOutgoingData(bool enable)
```

Comment:
Let's return error, we cannot support dynamic enabling of tx timestamping. We can only disable it dynamically.
In order to implement dynamic enablement of TX timestamping we need to support `SOF_TIMESTAMPING_OPT_ID_TCP` (which is not available for RHEL8).   https://docs.kernel.org/networking/timestamping.html

Another option - ensure that nothing has been sent through the socket and allow dynamic turning on only in this case. If we decide to do so -> we need to to add a TC for  that later.

Reply:
I will implement option 2) as you describe. Done.